src/processTargets/processTargets.ts

```diff
@@ -229,7 +231,13 @@ export function processModifierStages(
 }
 
 function calcIsReversed(anchor: Target, active: Target) {
-  return anchor.contentRange.start.isAfter(active.contentRange.start);
+  if (anchor.contentRange.start.isAfter(active.contentRange.start)) {
+    return true;
+  }
+  if (anchor.contentRange.start.isBefore(active.contentRange.start)) {
+    return false;
+  }
+  return anchor.contentRange.end.isAfter(active.contentRange.end);
```

Comment:
how come this had to change?

Reply:
This was incorrectly implemented, but only for very niche cases. If I remember correctly it had to do with reverse selection `this past start of token` was inconsistent with `this past end of token`